### PR TITLE
Rogue planets

### PR DESCRIPTION
**Pull request by Gutin1**

### CODE REVIEW

**Comment by Peter-Crawley** [comment]
Rebase required.

File: src/main/kotlin/net/starlegacy/feature/space/Orbits.kt [hunk 1]
```diff
@@ -10,9 +10,9 @@ import net.starlegacy.util.blockplacement.BlockPlacement
 object Orbits : SLComponent() {
 	// schedule orbiting all the planets every midnight
 	override fun onEnable() {
-//		Tasks.sync {
-//			orbitPlanets()
-//		}
+		Tasks.sync {
+			orbitPlanets()
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/feature/space/Orbits.kt:14]: Why reenable this?

File: src/main/kotlin/net/starlegacy/database/schema/space/Planet.kt [hunk 2]
```diff
@@ -7,6 +7,7 @@ import net.starlegacy.database.DbObject
 import net.starlegacy.database.Oid
 import net.starlegacy.database.OidDbObjectCompanion
 import net.starlegacy.database.objId
+import org.dynmap.Client.Update
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/database/schema/space/Planet.kt:10]: I can't tell where (if at all) this import is used.

File: src/main/kotlin/net/starlegacy/command/space/PlanetCommand.kt [hunk 3]
```diff
@@ -211,9 +246,17 @@ object PlanetCommand : SLCommand() {
 		val spaceWorld = planet.spaceWorld ?: throw InvalidCommandArgument("That planet's space world isn't loaded!")
 
 		val elapsedNanos = measureNanoTime {
+			if(planet.rogue) {
+				planet.setLocation(true)
+				spaceWorld.save()
+				SpaceMap.refresh()
+			}
+
+			if(!planet.rogue) {
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/command/space/PlanetCommand.kt:255]: else

File: src/main/kotlin/net/starlegacy/command/space/PlanetCommand.kt [hunk 4]
```diff
@@ -211,9 +246,17 @@ object PlanetCommand : SLCommand() {
...
 			planet.orbit(true)
 			spaceWorld.save()
```
Comment (Peter-Crawley) [inline src/main/kotlin/net/starlegacy/command/space/PlanetCommand.kt:257]: Move outside of the if statement as it is called regardless.

**Review by Peter-Crawley — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/starlegacy/feature/space/Orbits.kt (hunk 1 above)
Comment (Gutin1) [reply src/main/kotlin/net/starlegacy/feature/space/Orbits.kt:14]: keep hearing people ask when its coming back, we probably have the server resources for it now

**Pull request merged**